src/module.c

```diff
@@ -2204,6 +2204,17 @@ int RM_ReplyWithDouble(RedisModuleCtx *ctx, double d) {
     addReplyDouble(c,d);
     return REDISMODULE_OK;
 }
+/* Send a string as a BigNumber reply.  
```

Comment:
```suggestion

/* Send a string as a BigNumber reply.  
```
Missing a blank line.